### PR TITLE
fix(RHINENG-11503): Playbook log doesn't display properly

### PR DESCRIPTION
**Pull request by johnsonm325**
Playbook log doesn't display properly after inital load.

To repro:
- Automation Toolkit > Remediations > A playbook  > Activity > The data that the playbook ran successfully > Direct connected
- At this point the playbook log will flash on screen and disappear
- Click on the hostname

The issue was that playbookRunSystemDetails is a redux variable that remains after initial load. So when the playbook log is loaded again, the log viewer somehow only captures part of the string or none at all.

The best way to handle this issue is to clear the variable in redux on inital load to ensure there are no incorrect states.

### CODE REVIEW

File: src/components/ExecutorDetails/ExecutorDetails.js [hunk 1]
```diff
@@ -46,25 +45,34 @@ import columns from './Columns';
 import { useGetEntities } from './helpers';
 
 let refreshInterval;
+const _ = require('lodash');
```
Comment (bastilian) [inline src/components/ExecutorDetails/ExecutorDetails.js:48]: Since we only need `isEmpty` we could use `import isEmpty from 'lodash/isEmpty'`.

**Comment by AsToNlele** [comment]
@johnsonm325 do you have an account I could test this on?

**Comment by johnsonm325** [comment]
@AsToNlele It does not have to be a playbook that has a successful run. It just needs to have a playbook log.

**Review by AsToNlele — APPROVED** [review]
lgtm, thanks for writing a test! 😄

**Pull request merged**